### PR TITLE
Improvement/dispatcher doc

### LINKED ISSUE #139: Create CLI to send successful task email
It is necessary to create a CLI that based on a SAPS task with success in processing and archived, use the path of the backend dispatcher to make it send an email with links to access the task data. This includes other things like the Openstack Object Store Service Key which must be properly configured and thereby improve the dispatcher's documentation to ensure this configuration. Besides that it is also necessary for the current version of sending email, to enable the "less secure app access" setting of the domain of the noreply email used, so you must document how to do it.

### LINKED ISSUE #145: Create CLI to get task links
As mentioned by @thiagomanel  in the closed issue #139, the idea is to create a CLI that is possible to obtain the links to download the files for a given task. The input format would be something like:

```bash
bash get-task-links <user email> <user password> <task id> <dispatcher address>
```

and output something like:
```json
[
  {
    "name": "file01",
    "url": "link-to-download-file01"
  },
  {
    "name": "file02",
    "url": "link-to-download-file02"
  }
]
```

### PR DESCRIPTION
**Pull request by thiagoyeds**
Partially resolved #145

### CODE REVIEW

File: docs/dispatcher-install.md [hunk 1]
```diff
@@ -72,6 +65,66 @@ Note: ```-auth-version``` is the version of the deployed Openstack Identity Serv
 }
 ```
 
+- The `$noreply_email` and `$noreply_password` will be used by the SAPS deploy to send email notifications, however, in the current version, it is necessary to enable a setting that allows access to the less secure app, so do the following steps:
```
Comment (raonismaneoto) [inline docs/dispatcher-install.md]: ...to send email notifications. However...

File: docs/dispatcher-install.md [hunk 2]
```diff
@@ -72,6 +65,66 @@ Note: ```-auth-version``` is the version of the deployed Openstack Identity Serv
...
+
+### Gmail
+
+Sign in to your Gmail account and go to **Manage your Google account**
+![Manage your Google account](img/dispatcher-install-configure-noreply-email-img1.png)
+
+After, access **Security** settings
```
Comment (raonismaneoto) [inline docs/dispatcher-install.md]: Then, navigate to Security settings... I guess it sounds better

**Review by raonismaneoto — CHANGES_REQUESTED** [review]

File: docs/dispatcher-install.md (hunk 2 above)
Comment (thiagoyeds) [reply docs/dispatcher-install.md]: Done

File: docs/dispatcher-install.md (hunk 1 above)
Comment (thiagoyeds) [reply docs/dispatcher-install.md]: Done

File: docs/dispatcher-install.md [hunk 3]
```diff
@@ -72,6 +65,66 @@ Note: ```-auth-version``` is the version of the deployed Openstack Identity Serv
...
+![Manage your Google account](img/dispatcher-install-configure-noreply-email-img1.png)
+
+Then, navigate to **Security** settings
+![Security settings](img/dispatcher-install-configure-noreply-email-img2.png)
+
+Look for **Less secure app access** and enable it
+![Less secure app access](img/dispatcher-install-configure-noreply-email-img3.png)
+
+- The `$openstack_object_store_service_key` property in [Dispatcher configuration file](/config/dispatcher.conf) is used to access Object Store. After creating a new key, it must be configured using the command:
+
```
Comment (thiagomanel) [inline docs/dispatcher-install.md]: lines 82-87 seems to be on the wrong place

**Review by thiagomanel — CHANGES_REQUESTED** [review]
almost done

File: docs/dispatcher-install.md (hunk 3 above)
Comment (thiagoyeds) [reply docs/dispatcher-install.md]: Done

**Review by thiagomanel — APPROVED** [review]
lgtm

**Pull request merged**